code/core/template/stories/module-mocking.stories.ts

```diff
@@ -1,28 +0,0 @@
-import { global as globalThis } from '@storybook/global';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore This alias is set in the sandbox. Using ts-ignore instead of ts-expect-error to avoid build errors in the sandbox.
-
-import { foo } from '#utils';
```

Comment:
Why are these files removed?

Reply:
They were moved by norbert to the core directory, when he tried to consolidate the test package.

However, in the end the test package was not consolidated, so they were duplicated.